modules/apps/client-extension/client-extension-type-impl/src/main/java/com/liferay/client/extension/type/internal/configuration/CETConfigurationFactory.java

```diff
@@ -136,6 +136,21 @@ protected void deactivate() {
 								companyId));
 					}
 
+					if (Objects.equals(
```

Comment:
Why did this block get moved above the log message?

Reply:
Fixed